src/mpl2/src/object.h

```diff
@@ -361,17 +361,17 @@ class HardMacro
   void setX(float x);
   void setY(float y);
   const std::pair<float, float> getLocation() const;
-  float getX() const;
-  float getY() const;
+  inline float getX() const {return x_;}
```

Comment:
The inline keyword is unnecessary in this context, https://en.cppreference.com/w/cpp/language/inline:
```
A function defined entirely inside a class/struct/union definition, whether it's a member function or a non-member friend function, is implicitly an inline function
```

Please remove throughout this header

Reply:
Done.